src/redux/books/books.js

```diff
@@ -13,7 +13,6 @@ export const addBook = (payload) => async (dispatch) => {
     payload, // payload: payload(argument)
   });
 };
-// You can implement this one according to addBook
```

Comment:
- [ ] Please your app is breaking, kindly fix that and resubmit for review. See image below.

![app_failing](https://user-images.githubusercontent.com/65852621/146356887-21c6f705-cc56-4882-ae60-8039af4f91a9.png).

Reply:
i don't understand how it is breaking, i can't recreate the problem. can you share how you got to that point?